Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/SqlServer/Sql2016/SqlStorageObjectFactoryTest.cs

```diff
@@ -127,7 +128,7 @@ public void SetUp ()
     [Test]
     public void CreateStorageProvider ()
     {
-      var result = _sqlStorageObjectFactory.CreateStorageProvider(_rdbmsProviderDefinition, _persistenceExtensionStub.Object);
+      var result = (StorageProvider)_sqlStorageObjectFactory.CreateStorageProvider(_rdbmsProviderDefinition, _persistenceExtensionStub.Object);
```

Comment:
note, the ticket requests inlining the StorageProvider if possible (either in this task or a separate task). there's no ticket on this so I assume it's missing in this task or not possible. Please update info on this :)